lib/config/schema.json

```diff
@@ -355,6 +355,12 @@
       "http_method": "post",
       "url": "https://api.appstoreconnect.apple.com/v1/inAppPurchaseAppStoreReviewScreenshots"
     },
+    {
+      "alias": "commit_in_app_purchase_review_screenshot",
```

Comment:
I'd like to start standardizing on the naming of these aliases so that in the future we can autogenerate these aliases instead of having to provide them in this schema. To make this future compatible can we name this `update_in_app_purchase_app_store_review_screenshot`?

Reply:
I see! Sure, we can do it! We took the name from Apple's documentation and it made sense in the context

Reply:
Addressed in 6c9b18b